iosApp/iosApp/Pages/Map/HomeMapViewHandlerExtension.swift

```diff
@@ -205,23 +208,27 @@ extension HomeMapView {
         })
         let stopSequence = trip?.stopSequence?.intValue
 
-        guard let routeId = vehicle.routeId ?? trip?.trip.id else {
+        guard let routeId = trip?.trip.routeId ?? vehicle.routeId else {
             // TODO: figure out something to do if this is nil
             return
         }
+        let routeType = globalData?.routes[routeId]?.type
```

Comment:
Oh, sorry, I was unclear. I was thinking that it would be better to do this by passing the route ID into `TripDetailsPage`, then get the route from global to check the route type there, so that we wouldn't need to include the route type in the navigation entry at all. It seems redundant to include a piece of information that's derived from a different piece of information that's already included. And since changing the nav items has a relatively broad reach, I think it's best to only put strictly necessary and unique filter information in there.

Reply:
Oh, got it! I'll make that change 😄